### PR TITLE
Move mapping and Add monthly Columns for Quarterly

### PR DESCRIPTION
**Pull request by tmccoy529**
This PR address both #85 and also a redesign of the quarterly confirmation page.

### CODE REVIEW

File: src/data/TaxReturnMapper.js [hunk 1]
```diff
@@ -23,6 +28,89 @@ const GetDataForMonth = (taxReturn, monthIndex) => {
...
+
+    monthSubmitted = monthSubmitted.concat(
+      GetFormatedDateTime(
+        new Date(MonthlyData[i].Month + "/01/" + MonthlyData[i].Year),
+        dateFormat
+      ) + " "
+    );
+  }
+
+  if (ReturnType === 1) {
```
Comment (martypowell) [inline src/data/TaxReturnMapper.js]: this line is particularly concerning, in the case that the database values can change, can we check the length of the data instead?

Comment (martypowell) [reply src/data/TaxReturnMapper.js]: probably object.keys of the months or something alogn those lines?

File: src/data/TaxReturnMapper.js [hunk 2]
```diff
@@ -23,6 +28,89 @@ const GetDataForMonth = (taxReturn, monthIndex) => {
...
+    monthlyOccupancy = monthlyOccupancy.concat("$" + totalMonthlyOccupancy);
+    monthlyExemption = monthlyExemption.concat("$" + totalMonthlyExemption);
+    monthlyPenalty = monthlyPenalty.concat("$" + totalMonthlyPenalty);
+    monthlyRemittedTax = monthlyRemittedTax.concat(
+      "$" + totalMonthlyRemittedTax
+    );
+    monthSubmitted = monthSubmitted.concat("Totals");
+  }
+
+  const monthOfReturn = ReturnType === 1 ? 2 : 0;
```
Comment (martypowell) [inline src/data/TaxReturnMapper.js]: same applies here for return type

Comment (martypowell) [reply src/data/TaxReturnMapper.js]: i picutre something like

```js
const isMontly = Object.keys(total).length === 1;
```

Comment (tmccoy529) [reply src/data/TaxReturnMapper.js]: done

File: src/data/TaxReturnMapper.js (hunk 1 above)
Comment (tmccoy529) [reply src/data/TaxReturnMapper.js]: done

**Comment by martypowell** [comment]
@tmccoy529 noticed a couple things

- Can we make date header's bold
- The Bottom Right Total Bold
- Add confirmation number below "Transient Tax Return Submitted"

See here: http://staging.baltimorecountymd.gov/TransientTax/confirmation-quarterly.html

I think well probalby also want to remove the confimration number in front of Transient Occupancy Tax Return Details: but i haven't gotten confirmation

File: src/data/TaxReturnMapper.js [hunk 3]
```diff
@@ -23,6 +34,135 @@ const GetDataForMonth = (taxReturn, monthIndex) => {
...
+
+  var totalMonthlyOccupancy = 0;
+  var totalMonthlyExemption = 0;
+  var totalMonthlyPenalty = 0;
+  var totalMonthlyRemittedTax = 0;
+  var totalMonthlyInterest = 0;
+  var totalMonthlyTaxCollected = 0;
+  var totalMonthlyNetRoomRental = 0;
+
+  for (var i = 0; i < MonthlyData.length; i++) {
```
Comment (martypowell) [inline src/data/TaxReturnMapper.js]: per the original comment, we shoudl really clean this up. We will want to address these along with https://github.com/baltimorecounty/react-transient-tax/issues/88.

**Pull request merged**